modules/Link.js

```diff
@@ -46,17 +46,24 @@ class Link extends React.Component {
   }
 
   handleClick = (event) => {
+
     if (this.props.onClick)
       this.props.onClick(event)
 
-    if (
-      !event.defaultPrevented && // onClick prevented default
-      !this.props.target && // let browser handle "target=_blank" etc.
-      !isModifiedEvent(event) &&
-      isLeftClickEvent(event)
-    ) {
-      event.preventDefault()
-      this.context.router.transitionTo(this.props.to)
+    if (this.context.router) {
+      if (
+        !event.defaultPrevented && // onClick prevented default
+        !this.props.target && // let browser handle "target=_blank" etc.
+        !isModifiedEvent(event) &&
+        isLeftClickEvent(event)
+      ) {
+        event.preventDefault()
+        this.context.router.transitionTo(this.props.to)
+      }
+    } else {
```

Comment:
just drop the else, most likely we're just in a unit test and we don't want warnings anyway.